CLDConfig/CaloDigi/CaloDigi.py

```diff
@@ -19,6 +19,7 @@
 from Gaudi.Configuration import WARNING
 from Configurables import MarlinProcessorWrapper
 
+import sys
```

Comment:
why is this needed?

Reply:
I added a `sys.exit(1)` below if the integration time is not 10 or 400 ns. Probably not needed since k4run has this import but my editor (and everyone's else if configured) complains that it's not imported.

Reply:
Oops I am just blind lol